Configurations/10-main.conf

```diff
@@ -1345,6 +1345,7 @@ sub vms_info {
             my @ex_libs = ();
             push @ex_libs, 'ws2_32.lib' unless $disabled{sock};
             push @ex_libs, 'gdi32.lib advapi32.lib crypt32.lib user32.lib';
+            push @ex_libs, 'ncrypt.lib';
```

Comment:
I think it can be argued that `crypt32.lib` shouldn't be there with the argument that only `capi.dll` needs it as well, no?  I was pondering another variant, to add language in `build.info` to specify extra external libraries to be used with specific programs or shared libraries.  Would it be worth it?  Maybe.  For now, though, I'll use the pragma, thanks for pointing it out.